server/server.go

```diff
@@ -10,13 +10,22 @@ import (
 	logging "github.com/ipfs/go-log/v2"
 	"github.com/ipni/go-libipni/apierror"
 	finderhttpclient "github.com/ipni/go-libipni/find/client/http"
+	"github.com/ipni/go-libipni/find/model"
 	"github.com/ischasny/dhfind/metrics"
 	"go.uber.org/zap"
 )
 
-// preferJSON specifies weather to prefer JSON over NDJSON response when request accepts */*, i.e.
-// any response format, has no `Accept` header at all.
-const preferJSON = true
+const (
+	// preferJSON specifies weather to prefer JSON over NDJSON response when request accepts */*, i.e.
+	// any response format, has no `Accept` header at all.
+	preferJSON = true
+
+	// methodMultihash represents a multihash method for latency reporting purposes
```

Comment:
This sounds like a ChatGPT comment.  Copilot?

Reply:
I think this one was me 😢